airflow/cli/commands/connection_command.py

```diff
@@ -269,7 +269,7 @@ def connections_add(args):
             msg = msg.format(
                 conn_id=new_conn.conn_id,
                 uri=args.conn_uri
-                or urlunparse(
+                or urlunsplit(
```

Comment:
I don’t think this works, `urlunparse` takes a six-item iterable, but `urlunsplit` takes a five-item one. More review is needed.

Reply:
I fixed this now